src/languages/en.ts

```diff
@@ -1029,6 +1029,7 @@ const translations = {
         categorySelection: 'Select a category to better organize your spend.',
         error: {
             invalidCategoryLength: 'The category name exceeds 255 characters. Please shorten it or choose a different category.',
+            invalidTagLength: 'The tag name exceeds 255 characters. Please shorten it or choose a different tag.',
```

Comment:
Works well!
One small thing, I propose we improve this a little bit, it's not right to have 255 hardcoded into the translation string here

```suggestion
            invalidTagLength: ({limit}: CharacterLimitParams) => `The tag name exceeds ${limit} characters. Please shorten it or choose a different tag.`,
```